### DISCUSSION TITLE
layout breakdown in chat service

### DISCUSSION THREAD

**Discussion by Broninn** [LiveHelperChat/livehelperchat#2115] [Q&A]
Hello everything is fine?

LHC Version 4.39
We are facing the following situation. We have 2 bots in 2 different departments. 1 to 1.
When performing a service in BOT 1, department 1, the service layout is OK, as shown in the print below.

![CHAT ok](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/22ab69de-f09a-4cbc-9d79-c49252ab2682)

When serving BOT 2, department 2, the layout breaks, as shown in the print below.

![CHAT wrong](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/b536f371-be7c-46a1-a314-d42b91ca4f54)

Analyzing the HTML, the div div class="col-md-4 chat-main-right-column mh150" translate="no"  simply does not exist. It is highlighted in the prints.

I looked in the theme settings, admin themes, departments, bot, and didn't find anything that indicated this problem. Do you have any ideas or suggestions for adjustments?

**Comment by remdex** [comment]
ping me directly on discord i might look directly to Your back office.

**Comment by Broninn** [comment]
Ok, I'll take all informations with the IT team, and I call you.

**Comment by remdex** [comment]
Alos looking at screenshot it can be that your bot renders invalid HTML if anywhere sutch element is used. Also your IT guys can just expoirt particular chat lh_msg records for that chat.

**Reply by Broninn** [reply]
HI, exactly. I checked all the HTML inserted via BOT, and there was a div that was not closed. Thank you for the idea and suggestion, and please help others who may be in this situation.
